### ISSUE TITLE
About TLP & auto-cpufreq compatibility

### ISSUE THREAD

**Issue by Dytique** [linrunner/TLP#724]
**The reason for this issue is an ambiguity in the documentation about the compatibility between TLP & auto-cpufreq**

During my researches about how to get the most out of the battery on my Linux laptop, I came across two great projects: TLP and auto-cpufreq. But, diving into the two projects, I realized that the possibility of running them in tandem was very ambiguous.
I'm not sure if this is the appropriate place to ask about this, but I haven't found any good answers anywhere else (even on the auto-cpufreq Discord server, where I met other users with the same goal).

In the [TLP docs](https://linrunner.de/tlp/faq/installation.html#does-tlp-conflict-with-other-power-management-tools), it says:

> auto-cpufreq: only impacts CPU settings that are not active in TLP’s default configuration. There will be no conflict as long as these remain deactivated.

But in the default config of TLP(UI), I see those CPU-related parameters enabled by default:
* **CPU_ENERGY_PERF_POLICY_ON_AC**
* **CPU_ENERGY_PERF_POLICY_ON_BAT**

I frankly don’t know what those settings do, but it seems to me that this has an influence on CPU behavior that could conflict with auto-cpufreq.

So I was wondering **if it would be possible to clarify this point in the documentation**, because I sincerely believe that it would be a huge gain for the whole Linux community to be able to use these two great tools in a complementary way.

**TL;DR**
It would be great to
* provide more information to the end user regarding the compatibility with auto-cpufreq

**Alternatives I've considered**
I've considered using TLP's CPU management features, but they require more advanced technical skills and manual configuration, and I'm afraid of damaging my hardware by fiddling with it.
On the other hand, I've considered uninstalling TLP, but I think it's a shame to part with the multitude of features TLP offers.

**Additional context**
Here's the initial post I made on the endeavourOS forum:
https://forum.endeavouros.com/t/tlp-auto-cpufreq-compatibllity/50431
